dotnet/website/articles/getting-start.md

```diff
@@ -1,31 +1,17 @@
-## AutoGen for .NET
-
+### Get start with AutoGen for dotnet
 [![dotnet-ci](https://github.com/microsoft/autogen/actions/workflows/dotnet-build.yml/badge.svg)](https://github.com/microsoft/autogen/actions/workflows/dotnet-build.yml)
+[![Discord](https://img.shields.io/discord/1153072414184452236?logo=discord&style=flat)](https://discord.gg/pAbnFJrkgZ)
+[![NuGet version](https://badge.fury.io/nu/AutoGen.Core.svg)](https://badge.fury.io/nu/AutoGen.Core)
 
-### Get start with AutoGen for dotnet
 Firstly, following the [installation guide](Installation.md) to install AutoGen packages.
 
 Then you can start with the following code snippet to create a conversable agent and chat with it.
 
 [!code-csharp[](../../sample/AutoGen.BasicSamples/CodeSnippet/GetStartCodeSnippet.cs?name=snippet_GetStartCodeSnippet)]
 [!code-csharp[](../../sample/AutoGen.BasicSamples/CodeSnippet/GetStartCodeSnippet.cs?name=code_snippet_1)]
 
-### Functionality
-- ConversableAgent
-    - [x] function call
-    - [x] code execution (dotnet only, powered by [`dotnet-interactive`](https://github.com/dotnet/interactive))
-
-- Agent communication
-    - [x] Two-agent chat
-    - [x] Group chat
-
-- [ ] Enhanced LLM Inferences
-
-- Exclusive for dotnet
-    - [x] Source generator for type-safe function definition generation
-
 ### Examples
 You can find more examples under the [sample project](https://github.com/microsoft/autogen/tree/dotnet/dotnet/sample/AutoGen.BasicSamples).
 
-### Update log
-[!INCLUDE [Update log](../../nuget/NUGET.md)]
\ No newline at end of file
+### Report a bug or request a feature
+You can report a bug or request a feature by creating a new issue in the [github issue](https://github.com/microsoft/autogen/issues).
```

Comment:
"And specifying label the label "donet" " ... 

perhaps we need a custom issue template for .NET related issues.